src/nnet3/nnet-combine.cc

```diff
@@ -504,15 +504,16 @@ double NnetCombiner::ComputeObjfAndDerivFromNnet(
                                             end = egs_.end();
   for (; iter != end; ++iter)
     prob_computer_->Compute(*iter);
-  const SimpleObjectiveInfo *objf_info = prob_computer_->GetObjective("output");
-  if (objf_info == NULL)
+  double tot_weights,
+    tot_objf = prob_computer_->GetTotalObjective(&tot_weights);
+  if (tot_objf == 0.0)
```

Comment:
This if-statement and the associated error message its prints are not suitable, this should not be an error.  Please remove both